packages/web/src/components/toast/ToastContext.tsx

```diff
@@ -96,8 +124,26 @@ export const ToastContextProvider = (props: { children: ReactNode }) => {
             content={item.content}
             link={item.link}
             linkText={item.linkText}
-            leftIcon={item.leftIcon}
-            rightIcon={item.rightIcon}
+            leftIcon={
+              item.leftIcon && (
+                <item.leftIcon
```

Comment:
If it's always receiving dismiss, do we just have x be the default and always pick a side?